src/util/ui.cpp

```diff
@@ -246,6 +266,18 @@ namespace util::ui {
         return createBadge(compute(data).badgeIcon);
     }
 
+    // CCMenu* createBadgeMenuIfSpecial(const SpecialUserData& data) {
+    //     if (!data.roles) return nullptr;
+
+    //     return createBadgeMenu(RoleManager::get().createRoleArray(data.roles.value()));
+    // }
+
+    // CCMenu* createBadgeMenuIfSpecial(const UserEntry& data) {
+    //     if (data.userRoles.empty()) return nullptr;
+
+    //     return createBadgeMenu(RoleManager::get().createRoleArray(data.userRoles));
+    // }
+
```

Comment:
preferably comment them out from the header as well, or remove completely if it's not needed, otherwise if someone tries to use them they will get a weird linker errors